File: code/get-set.java
void renderItem() {
    visitCount.setText(String.format(...), ...)
         
    if(item.cellPhone == null ||
       item.cellPhone.isEmpty()) {
        phone1.setVisibility(View.GONE)
    } else {
	  phone1.setVisibility(View.VISIBLE)
    }

    shopName.text = 
        item.name != null ? item.name : "..."
}

File: code/get-set.kt
fun renderItem(item: SomeItem) {
    visitCount.text = "${item.visitCount}次拜访"

    phone1.visibility = 
            if(item.cellPhone.isNullOrEmpty()) 
               View.GONE else View.VISIBLE

    shopName.text = item.name ?: "暂无商户名称"
}